RELEASING.md

```diff
@@ -199,6 +199,10 @@ Tagging the Release
     Push gh-pages to the main repository and verify the current version is
     [live on grpc.io](https://grpc.io/grpc-java/javadoc/).
 
+    ```bash
+    git push upstream gh-pages
```

Comment:
Combine this with the previous block and remove the "Push gh-pages to th emain repository"? (Keep "verify the current version...")

Reply:
Done. Kept the encouragement to verify.